decavision/model_training/tfrecords_image_classifier.py

```diff
@@ -310,14 +310,28 @@ def fit(self, save_model=None, export_model=None, min_accuracy=None,
             callbacks = [tensorboard_callback]
 
         # if we want to stop training when no sufficient improvement in accuracy has been achieved
-        if min_accuracy:
+        if min_accuracy is not None:
             early_stop = tf.keras.callbacks.EarlyStopping(monitor='val_sparse_categorical_accuracy',
                                                           min_delta=min_accuracy)
             if not callbacks:
                 callbacks = [early_stop]
             else:
                 callbacks.append(early_stop)
 
+        # use reduce learning rate and early stopping callbacks
+        else:
+            early_stop = tf.keras.callbacks.EarlyStopping(monitor='val_sparse_categorical_accuracy',
```

Comment:
Here my suggestion would be make the patience a parameter so people can decide it, and if it's 0 then not use earlystopping. Also this version seems better than the min_accuracy thing so I would just replace it. I think the reduceLR can be used all the time.

Reply:
Should I change the min_accuracy parameter to patience for early stopping? If I replace it, min_acc has no use, as it just adds early stopping callback.

Reply:
Yes you replace min_accuracy by patience. You basically replace all the code that concerns min_Accuracy with your code.

Reply:
Alright!

Reply:
Made the changes.